aws/resource_aws_elastic_beanstalk_environment_test.go

```diff
@@ -283,6 +287,49 @@ func TestAccAWSBeanstalkEnv_resource(t *testing.T) {
 	})
 }
 
+func TestAccAWSBeanstalkEnv_tags(t *testing.T) {
```

Comment:
This is currently failing with:
```
=== RUN   TestAccAWSBeanstalkEnv_tags
--- FAIL: TestAccAWSBeanstalkEnv_tags (474.54s)
    testing.go:513: Step 0 error: Check failed: Check 1/1 error: InvalidParameter: 1 validation error(s) found.
        - missing required field, ListTagsForResourceInput.ResourceArn.
```

The first test step needs to include this as the first check so `app` is correctly populated: `testAccCheckBeanstalkEnvExists("aws_elastic_beanstalk_environment.tfenvtest", &app),`

FYI you can run the acceptance test locally with `make testacc TEST=./aws TESTARGS='-run=TestAccAWSBeanstalkEnv_tags` to verify